tests/DocumentationExamplesTest.php

```diff
@@ -925,6 +925,29 @@ public function testExample_55_58()
         $this->assertInventoryCount(0);
     }
 
+    public function testExample_59_61()
+    {
+        $db = new Database($this->manager, $this->getDatabaseName());
+
+        // Start Changestream Example 1
+        $cursor = $db->inventory->watch();
+        $cursor->next();
+        $next = $cursor->current();
+        // End Changestream Example 1
+
```

Comment:
In between the example blocks, we can do assertions just to test that the examples perform as expected.

I realize all of these examples do nothing between `watch()` and iterating, so asserting empty results seems sufficient.